localstack/aws/api/sqs/__init__.py

```diff
@@ -0,0 +1,1298 @@
+    """A list of received messages."""
+
+    Messages: Optional[MessageList]
+
+
+class RemovePermissionRequest(ServiceRequest):
+    QueueUrl: String
+    Label: String
+
+
+class SendMessageBatchRequestEntry(TypedDict, total=False):
+    """Contains the details of a single Amazon SQS message along with an
+    ``Id``.
+    """
+
+    Id: String
+    MessageBody: String
+    DelaySeconds: Optional[Integer]
+    MessageAttributes: Optional[MessageBodyAttributeMap]
+    MessageSystemAttributes: Optional[MessageBodySystemAttributeMap]
+    MessageDeduplicationId: Optional[String]
+    MessageGroupId: Optional[String]
+
+
+SendMessageBatchRequestEntryList = List[SendMessageBatchRequestEntry]
+
+
+class SendMessageBatchRequest(ServiceRequest):
+    QueueUrl: String
+    Entries: SendMessageBatchRequestEntryList
+
+
+class SendMessageBatchResultEntry(TypedDict, total=False):
+    """Encloses a ``MessageId`` for a successfully-enqueued message in a
+    ``SendMessageBatch.``
+    """
+
+    Id: String
+    MessageId: String
+    MD5OfMessageBody: String
+    MD5OfMessageAttributes: Optional[String]
+    MD5OfMessageSystemAttributes: Optional[String]
+    SequenceNumber: Optional[String]
+
+
+SendMessageBatchResultEntryList = List[SendMessageBatchResultEntry]
+
+
+class SendMessageBatchResult(TypedDict, total=False):
+    """For each message in the batch, the response contains a
+    ``SendMessageBatchResultEntry`` tag if the message succeeds or a
+    ``BatchResultErrorEntry`` tag if the message fails.
+    """
+
+    Successful: SendMessageBatchResultEntryList
+    Failed: BatchResultErrorEntryList
+
+
+class SendMessageRequest(ServiceRequest):
+    QueueUrl: String
+    MessageBody: String
+    DelaySeconds: Optional[Integer]
+    MessageAttributes: Optional[MessageBodyAttributeMap]
+    MessageSystemAttributes: Optional[MessageBodySystemAttributeMap]
+    MessageDeduplicationId: Optional[String]
+    MessageGroupId: Optional[String]
+
+
+class SendMessageResult(TypedDict, total=False):
+    """The ``MD5OfMessageBody`` and ``MessageId`` elements."""
+
+    MD5OfMessageBody: Optional[String]
+    MD5OfMessageAttributes: Optional[String]
+    MD5OfMessageSystemAttributes: Optional[String]
+    MessageId: Optional[String]
+    SequenceNumber: Optional[String]
+
+
+class SetQueueAttributesRequest(ServiceRequest):
+    QueueUrl: String
+    Attributes: QueueAttributeMap
+
+
+TagKeyList = List[TagKey]
+
+
+class TagQueueRequest(ServiceRequest):
+    QueueUrl: String
+    Tags: TagMap
+
+
+class UntagQueueRequest(ServiceRequest):
+    QueueUrl: String
+    TagKeys: TagKeyList
+
+
+class SqsApi:
+
+    service = "sqs"
+    version = "2012-11-05"
+
+    @handler("AddPermission")
+    def add_permission(
+        self,
+        context: RequestContext,
+        queue_url: String,
+        label: String,
+        aws_account_ids: AWSAccountIdList,
+        actions: ActionNameList,
+    ) -> None:
+        """Adds a permission to a queue for a specific
```

Comment:
nit: Do we want to keep the docstrings in the service interface here? Not sure it adds too much value - something we can think of over time, as the number of ASF interface definitions grows... @thrau @baermat

Reply:
sorry,  read your comment too late! but good point, we can re-generate the api stubs without the docs :-)